virtwho/parser.py

```diff
@@ -9,116 +31,267 @@
 SAT5 = "satellite"
 SAT6 = "sam"
 
+# List of supported virtualization backends
+VIRT_BACKENDS = ['libvirt', 'vdsm', 'esx', 'rhevm', 'hyperv', 'xen']
```

Comment:
Might I suggest that this list be defined (or rather computed) via something in the Virt base class? (virtwho/virt/virt.py)
As each of these values exist as a class attribute on each subclass of Virt, and Virt already checks through all of these values in the "from_config" method, we might want to avoid having to update this list in two places should there be any additions or removals in this area